CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+# Code of Conduct
+
+Everyone interacting in the pyOpenSci project's codebases, issue trackers, chat rooms, and communication venues is
```

Comment:
I've been meaning to add an [organization wide COC](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) in our [.github repo](https://github.com/pyOpenSci/.github/tree/main) --  I just made an asana task for this as it doesn't belong to a single repo rather to the organization